tools/codegen/generate.py

```diff
@@ -1,915 +1,396 @@
 """Weave SDK Code Generator.
```

Comment:
Any idea what this notation is? `100755 -> 100644`

<img width="573" height="94" alt="Image" src="https://github.com/user-attachments/assets/2cef47e1-8f82-4f69-a16e-6f68ea7e8299" />